molecule/mtls-customcerts-rhel/molecule.yml

```diff
@@ -9,7 +9,7 @@ platforms:
     hostname: zookeeper1.confluent
     groups:
       - zookeeper
-    image: geerlingguy/docker-centos7-ansible
+    image: wadhwa1/docker-centos7:v2
```

Comment:
created custom docker image `wadhwa1/docker-centos7:v2` so as to first reproduce the cp-env issue via molecule tests and make our tests fail. 

Verified via cp-ansible-ondemand-job  https://jenkins.confluent.io/job/cp-ansible-on-demand/656/testReport/molecule.mtls-customcerts-rhel.molecule/yml/test/

TASK [confluent.platform.ssl : Verify Cert Chain] ******************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: No module named cryptography
fatal: [zookeeper1]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (cryptography >= 1.5) on zookeeper1.confluent's Python /usr/bin/python. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}

Reply:
I understand the motive, but do you want to submit these changes ? If thats the case, please move the images to our org registry in docker hub.

Reply:
There's another ticket in sprint board to create custom images, maintain respective dockerfiles, and host images onto our org registry. we can skip submitting these changes for now, I had added them solely for reproducing the scenario and testing changes. I'll revert these/ remove them from this changeset. sounds okay to you ?

Reply:
done, reverted this temp change